filenamify.js

```diff
@@ -37,4 +41,27 @@ const filenamify = (string, options = {}) => {
 	return string;
 };
 
+const recursivelyFilenamify = (string, options = {}) => {
```

Comment:
The function name should describe what it achieves, not how it works. yes, it recursively runs, but that's not all.

Reply:
OK, before I make a new commit and to save time for both of us, I have a few candidates for the new name

- `filenamifyStringContainingReplacement`
- `filenamifyStringContainingReplacementString`
- `filenamifyStringContainingTheReplacementString`
- `filenamifyInputStringContainingReplacementString`
- `filenamifyInputStringContainingTheReplacementString`
- `filenamifyUserInputStringContainingReplacementString`
- `filenamifyUserInputStringContainingTheReplacementString`

I'm not sure which might sound good to you, or may I sincerely ask you, if you will, to advise a new name for this function. Thank you so much.

Reply:
I think it can just be `handleReplacementStringInUserInput`.

The function needs a doc comment that explains the logic.